nodes/project-link.html

```diff
@@ -368,94 +368,87 @@
         <dd>
-            The message to send to the <code>target</code> project <code>path</code>
+            <p>The node will send the complete message object it receives.</p>\
+            <p>Due to the way messages are sent, not all types of property will
+                be included. For example, the <code>msg.req</code> and <code>msg.res</code> properties
+                used by the HTTP nodes will not be sent.
+            </p>
         </dd>
     </dl>
 </script>
 
 <script type="text/html" data-help-name="project link call">
-    <p>Send messages to other projects and get a response</p>
-    <h3>Properties</h3>
-    <dl class="message-properties">
-        <dt><i>timeout</i><span class="property-type">number</span></dt>
-        <dd>
-            The number of seconds to wait for the sent <code>msg</code> to be returned
-            before raising a <code>timeout</code> error
-        </dd>
-        <dt><i>target</i></dt>
-        <dd>
-            The project to send messages to
-        </dd>
-        <dt><i>path</i> <span class="property-type">string</span></dt>
-        <dd>
-            A free form, slash separated path that identifies this link. 
-            Example: a path of <code>sensor/temperature</code> would send messages to 
-            the <code>target</code> project where a link node has a matching path
-        </dd>
-    </dl>
-    <h3>Input</h3>
+    <p>Send messages to other projects with your FlowForge Team and get a response back</p>
+    <h3>Details</h3>
+    <p>
+        This node can be used to send messages to other projects and then wait
+        for a response to be sent back.
+    </p>
+    <p>
+        The node can be configured with a <code>timeout</code> for how long it
+        should wait for a response. If a response does not arrive, it will log
+        an error that can be caught with a Catch node
+    </p>
+    <p>
+        The node is configured with a <code>topic</code> to send on. This works
+        like an MQTT topic - allowing projects to send messages targetting different
+        subscribers.
+    </p>
+    <h3>Output</h3>
     <dl class="message-properties">
         <dt><i>msg</i> <span class="property-type">object</span></dt>
         <dd>
-            The message to send to the <code>target</code> project <code>path</code>
+            The message sent by another project to this node.
         </dd>
-    </dl>
-    <h3>Output</h3>
-    <dl class="message-properties">
-        <dt><i>msg</i> <span class="property-type">object</span></dt>
+        <dt><i>projectLink</i> <span class="property-type">object</span></dt>
         <dd>
-            The <code>msg</code> object sent from a Project Link Out node set to
-            <code>return</code> mode
+            This property contains information about the source of the message.
+            <ul>
+                <li><code>topic</code> - the topic the message was received on</li>
+            </ul>
         </dd>
     </dl>
 </script>
```

Comment:
```suggestion

</script>
```

Add new line for linting requirements

Reply:
I suspect you mean a newline *after* the `</script>` not before